data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -172,185 +268,625 @@
+                                    [/variable]
+                                [/not]
+
+                                [then]
+                                    {GENERIC_UNIT 2 (Orcish Warrior) 31 2}
+                                    {GENERIC_UNIT 2 (Wolf Rider) 31 1}
+                                    {GENERIC_UNIT 2 (Orcish Archer) 30 3}
+                                    {FIND_NEARBY (side=2,4) 31 2 99}
+                                [/then]
+                            [/if]
+
+                            {MOVE_UNIT id=Mortic $found_unit.x $found_unit.y}
+                            [message]
+                                speaker=$found_unit.id
+                                message= _ "Wha- hey!"
+                            [/message]
+                            [animate_unit]
+                                [filter]
+                                    id=Mortic
+                                [/filter]
+
+                                flag=attack
+                                [primary_attack]
+                                    range=melee
+                                [/primary_attack]
+                                [facing]
+                                    x,y=$found_unit.x,$found_unit.y
+                                [/facing]
+                                hits=yes
+                            [/animate_unit]
+                            {KILL id=$found_unit.id ANIMATE=yes}
+                            [message]
+                                speaker=Mortic
+                                message= _ "The Hunt is called!
+The orcs our prey!
+Flight of Mortic, we fight!"
+                            [/message]
+                            {VARIABLE drakes_rebelling yes} # used for S13 into text
+                        [/then]
+                    [/if]
+
+                    [kill]
+                        side=1
+                        {NOT_ACROSS_RIVER}
+                    [/kill]
+
+                    [delay]
+                        time=1000
+                    [/delay]
+
+                    [if]
                         [have_unit]
-                            id=Owaec
+                            side,upkeep,level,count=1,full,2-99,0
                         [/have_unit]
 
                         [then]
+                            # if no non-loyal escapees
                             [message]
-                                speaker=Gweddry
-                                message= _ "Good, we have escaped from the orcs. Now we must return to save Wesnoth."
+                                speaker,image=narrator,wesnoth-icon.png
+                                message= _ "Oh no, only you and your loyal heroes have managed to escape!
+
+You’re welcome to retry an earlier scenario, but don’t feel like you have to. The rest of the campaign was designed to be beatable even with only Gweddry, Dacyn, and Owaec (though some bonus objectives may be out of reach)."
```

Comment:
'designed to be won'

Reply:
I actually don't really like that this exists at all, to be honest…

Reply:
> I actually don't really like that this exists at all, to be honest…

Do you mean the "oh no" message, the "losing un-recalled units" mechanic, or the ability to win without recalls?  It's probably fine to remove the message if you think the scenario is better without it, but the other 2 would require major rebalancing.

Reply:
I'm talking about the message specifically. It's not a bad idea to convey this information somewhere, but I'm not sure I like dragging the player through the first wall with this narration of mechanics. Maybe it could be added to objectives? There is also a possibility of showing it only on easy mode, or only on hard mode, which doesn't seem _quite_ as bad.

Reply:
I think only on easy would make sense.